### DISCUSSION TITLE
How to add .a extention library file in Visual studio code

### DISCUSSION THREAD

**Discussion by vinayhssloki** [microsoft/vscode-discussions#1119] [Extension Development QnA]
Hello,

We created a project using the Renesas build utilities and Renesas Debug extension on the R7FA4M2AC3CFM microcontroller in Visual Studio Code.

Additionally, we created a static library project using e2 Studio IDE and generated the .a extension library file. Now, we need to add this library file to the aforementioned project.

Could anyone please tell me how to add this library file to that project?

Thanks and Regards,
Vinay

**Comment by gjsjohnmurray** [comment]
Please use the Report Issue option from the Help menu to raise this with the extension developer.

This forum is focused on extension developers, so is not the right place to report problems arising when using extensions.
